src/System.Collections.Concurrent/tests/System.Collections.Concurrent.Tests.csproj

```diff
@@ -98,6 +98,8 @@
     <Compile Include="$(CommonTestPath)\System\Collections\IEnumerable.Generic.Serialization.Tests.cs">
       <Link>Common\System\Collections\IEnumerable.Generic.Serialization.Tests.cs</Link>
     </Compile>
+    <Compile Include="ConcurrentBagTests.netstandard1.7.cs" />
```

Comment:
I believe this is what's been done elsewhere.  @weshaggard, is there new guidance?